src/actions/chatActions.js

```diff
@@ -0,0 +1,16 @@
+export const CHAT_ADD_MESSAGE = 'CHAT_ADD_MESSAGE';
+export const CHAT_SET_USER_STATUS = 'CHAT_SET_USER_STATUS';
+export const CHAT_CLEAR_MESSAGES = 'CHAT_CLEAR_MESSAGES';
+export const CHAT_SET_USERNAME = 'CHAT_SET_USERNAME';
```

Comment:
Actually, i have not seen this kind of naming convention. Usually its just `ADD_MESSAGE` without any prefix. But for me it kind of makes more sense to have a sort of "namespace" to make it easier to se where an action comes from when debugging. 

If you are using `redux dev tools` then you can observe all the actions dispatched and its resulting effect on the state. After a while when the application matures and the codebase grows there will be a lot of different actions dispatched all the time. I predict it will be easier to look over the list of dispatched actions if it has a recognizable prefix.